cpp/unreal_plugins/SimulationController/Source/SimulationController/ImitationLearningTask.cpp

```diff
@@ -52,15 +46,17 @@ ImitationLearningTask::ImitationLearningTask(UWorld* world)
     actor_hit_event_component_->RegisterComponent();
     actor_hit_event_handle_ = actor_hit_event_component_->delegate_.AddRaw(this, &ImitationLearningTask::actorHitEventHandler);
 
-    // If the start/goal positions are not randomly generated, get them from a file
-    if (!Config::get<bool>("SIMULATION_CONTROLLER.IMITATION_LEARNING_TASK.GET_POSITIONS_FROM_TRAJECTORY_SAMPLING")) {
+    // Get start and end goal locations from a file
```

Comment:
- Don't make this file loading behavior optional. Make it mandatory, i.e., not wrapped in an if-statement.
- You can even inline the logic to load episodes directly into the constructor if you prefer. There is so little happening in this class now, so it won't be too confusing.
- The usage of "initial" vs "init" vs "start" vs "end" vs "goal" in this file is inconsistent and ugly. Pick a convention and stick to it throughout this entire class and the CSV file and the example code. I like "initial" and "goal".

Reply:
Updated this.